tests/ref/simulation/simulation_json_with_multi_constructor_used.json

```diff
@@ -353,24 +353,6 @@
             "value": 272.23520464657025
         }
     },
-    "outputs": [
-        {
-            "frequency": -1,
-            "frequency_offset": 0,
-            "name": "Surface output 1",
-            "output_fields": {
-                "items": [
-                    "Cp",
-                    "yPlus",
-                    "Cf",
-                    "CfVec"
-                ]
-            },
-            "output_format": "paraview",
-            "output_type": "SurfaceOutput",
-            "write_single_file": false
-        }
-    ],
     "time_stepping": {
```

Comment:
I think this will be a problem when we release python. Currently, user doesn't need to specify surface outputs explicitly. But after this PR, using python user needs to specify it which is different experience than in the past. I would suggest we address this issue with this PR or soon after.

cc: @benflexcompute

Reply:
We can add a `SurfaceOutput` instance if user did not specify during `SimulationParam.preprocess()` function. And populate all boundaries.

Reply:
The real question is do user rely on these auto-added surface output or not?

Reply:
I thought we didn't want SurfaceOutput as default for the python client. On webUI the user will have the option to remove this default, but not with python client. We didn't use to always generate SurfaceOutput with v1 python or json.

Reply:
this is a good point. How do you remove surface output at all? Maybe we can have a custom default constructor for outputs?

Reply:
@maciej-flexcompute For v1 python and json if surfaceOutput is empty we generate surface output for all surfaces with empty fields. After discussing with @benflexcompute we think it's unnecessary to replicate this behavior. The user experience should be consistent since output with empty fields is meaningless.